test/ui_test_helper.rb

```diff
@@ -348,6 +348,7 @@ def accept_alert
   # load file either in browser or download and check for existence
   def download_file(link, basename)
     link.click
+    sleep(3)
```

Comment:
I assume using a "waitUntil" construct is infeasible here.  Not a significant concern for a one-off, but we should revisit if this apparent need becomes more widespread or ends up causing false-positive failures.

Reply:
I agree - I hate adding sleep(n) calls, but the issue is that if there is a problem and the file doesn't load, the test will hang indefinitely.  I think a future refactor can have a while loop with a counter, but it just didn't seem like that much of an issue.